rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-wan-n6.yaml

```diff
@@ -0,0 +1,26 @@
+---
+###############################################################################
+# SR-IOV Network Definition for the NS of the workload
+#  n6 + vlan 901 + core_sriov_netdevice
+###############################################################################
+#
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: n6wan
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: core_sriov_netdevice
+  networkNamespace: open5gs-core
+  vlan: 901
```

Comment:
L15 vlan 901 is also not defined anywhere in the lab

Reply:
Same argument as above about the need of adding more IP nets and VLANs.